### PR TITLE
Fixed signup form

### LINKED ISSUE #1426: Implement signup page similar to chat
**Actual Behaviour**
Signup form is different from chat.susi signup form

**Expected Behaviour**
Must be similar for consistency.

**Screenshots of the issue**
Skills:

Chat:

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by love-san**
Fixes #1426 
Changes: Fixed signup form

Surge Deployment Link: https://pr-1462-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:
After commit:
susi2
Before commit:
susi3

### CODE REVIEW

File: src/components/Auth/SignUp/SignUp.js [hunk 1]
```diff
@@ -219,39 +240,55 @@ export default class SignUp extends Component {
...
             />
           </div>
           <div>
             <PasswordField
               name="password"
               style={fieldStyle}
+              inputStyle={inputpassStyle}
               value={passwordValue}
+              placeholder="Password"
+              underlineStyle={{ display: 'none' }} //
```
Comment (DNS-404) [inline src/components/Auth/SignUp/SignUp.js]: Remove `//`.

File: src/components/Auth/SignUp/SignUp.js [hunk 2]
```diff
@@ -219,39 +240,55 @@ export default class SignUp extends Component {
...
             <TextField
               name="email"
+              type="email"
               value={email}
+              className="textFields"
               onChange={this.handleChange}
+              style={emailStyle}
+              inputStyle={inputStyle}
+              labelText={{ color: '#878faf' }}
+              underlineStyle={{ display: 'none' }} //
```
Comment (DNS-404) [inline src/components/Auth/SignUp/SignUp.js]: Remove `//`.

File: src/components/Auth/SignUp/SignUp.js [hunk 3]
```diff
@@ -201,7 +189,40 @@ export default class SignUp extends Component {
   handleLogin = () => this.props.updateAuthDialog('login');
 
   render() {
-    const { fieldStyle, fontStyle, underlineFocusStyle } = styles;
+    const emailStyle = {
```
Comment (DNS-404) [inline src/components/Auth/SignUp/SignUp.js]: Don't write CSS of individual elements in separate objects. Rather, make a `styles` object at the end of the file and add CSS of every element in there. See how it's done in other components.

File: src/components/Auth/Login/Login.js [hunk 4]
```diff
@@ -45,12 +45,14 @@ const styles = {
   inputStyle: {
     height: '35px',
     marginBottom: '10px',
+    webkitTextFillColor: 'unset',
```
Comment (anshumanv) [inline src/components/Auth/Login/Login.js]: I believe this is only for chrome and safari? Please add the relevant css for other browsers.

File: src/components/Auth/Login/Login.js [hunk 5]
```diff
@@ -45,12 +45,14 @@ const styles = {
...
   },
   inputPasswordStyle: {
     height: '35px',
     marginBottom: '10px',
     marginRight: '50px',
     width: '90%',
+    webkitTextFillColor: 'unset',
```
Comment (anshumanv) [inline src/components/Auth/Login/Login.js]: same

File: src/components/Auth/SignUp/SignUp.js [hunk 6]
```diff
@@ -201,7 +189,39 @@ export default class SignUp extends Component {
   handleLogin = () => this.props.updateAuthDialog('login');
 
   render() {
-    const { fieldStyle, fontStyle, underlineFocusStyle } = styles;
+    const styles = {
```
Comment (anshumanv) [inline src/components/Auth/SignUp/SignUp.js]: Please don't keep this inside the render function, refer comment by @DNS-404

File: src/components/Auth/SignUp/SignUp.js [hunk 7]
```diff
@@ -201,7 +189,39 @@ export default class SignUp extends Component {
...
+        height: '35px',
+        marginBottom: '10px',
+        webkitTextFillColor: 'unset',
+      },
+      inputpassStyle: {
+        height: '35px',
+        marginBottom: '10px',
+        marginRight: '50px',
+        width: '90%',
+        webkitTextFillColor: 'unset',
```
Comment (anshumanv) [inline src/components/Auth/SignUp/SignUp.js]: similar argument

File: src/components/Auth/SignUp/SignUp.js [hunk 8]
```diff
@@ -219,51 +239,57 @@ export default class SignUp extends Component {
           <div>
             <TextField
               name="email"
+              type="email"
               value={email}
+              className="textFields"
```
Comment (anshumanv) [inline src/components/Auth/SignUp/SignUp.js]: Is this class in use anywhere?

File: src/components/Auth/SignUp/SignUp.js [hunk 9]
```diff
@@ -201,7 +189,39 @@ export default class SignUp extends Component {
...
+        border: '1px solid #ced4da',
+        fontSize: 16,
+        padding: '0px 10px',
+        width: '250px',
+        marginTop: '10px',
+      },
+      inputStyle: {
+        height: '35px',
+        marginBottom: '10px',
+        webkitTextFillColor: 'unset',
```
Comment (anshumanv) [inline src/components/Auth/SignUp/SignUp.js]: similar argument

**Pull request merged**